### PR TITLE
Initialize application

### PR DESCRIPTION
**Pull request by Sayed94h**
@razvanbrb 
Hey, 
I finished the initialize-application part of the project.
It does not look 100% the same as the Pomofocus website but I worked a lot on this part(yesterday night till now)

You can change the style and everything if you want to make it better

Have a great day!

### CODE REVIEW

File: project-planning/development-strategy.md [hunk 1]
```diff
@@ -126,43 +138,41 @@ __Assigned to Anna__
...
 ### Create a deleteItems method
+
 ---
 
 ## 6. User story : Edit task
 
-__As a user I should be able to delete the tasks that I want__
-
-__Assigned to Murrielle__
+**As a user I should be able to delete the tasks that I want**
```
Comment (julia-sod) [inline project-planning/development-strategy.md:156]: I guess Murrielle's story is to edit tasks, not to delete. The code looks very nice to me.

**Review by razvanbrb — APPROVED** [review]
I can see that your have work a lot. Great job @Sayed94h.

File: project-planning/development-strategy.md (hunk 1 above)
Comment (razvanbrb) [reply project-planning/development-strategy.md:156]: Yes that's true. My mistake. It was late when I did the dev-strategy. So you can edit on your branch please. Thank you!

Comment (julia-sod) [reply project-planning/development-strategy.md:156]: @razvanbrb Sure, I will do it)

**Comment by Sayed94h** [comment]
@razvanbrb I see that you approved the PR. So, if you would like please merge the PR without waiting for Anna because I think that she would be busy. I want to check it on the live page and then start with the set-time branch.

**Comment by razvanbrb** [comment]
Yes. I have checked but only in the Files change not live version. I don't have the link where I can check it live without merge it. This I still have to learn. I will merge it @Sayed94h .

**Pull request merged**